src/app/(homepage)/page.tsx

```diff
@@ -41,6 +42,11 @@ const Home = () => {
         <div>
           <Button type="primary" label="Create your first poll" />
         </div>
+        <div>
+          <div>
+            <Survey title="What do you think of this website?" lowest="It hurts my eyes!" highest="Most beautiful site ever!" />
+          </div>
+        </div>
```

Comment:
Is there any particular reason why this `Survey` component has been wrapped in two unstyled `<div>` elements? One would think/assume that the `Survey` has to be a block element, but since it's positioned absolutely, it breaks out of the normal document flow.